src/components/Dropdown/Dropdown.tsx

```diff
@@ -1,45 +1,67 @@
 import React, { useState } from 'react';
```

Comment:
If we need possibility to manually close Dropdown - maybe it will be better to just add public `close` method which you can call using ref?